core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -103,13 +104,12 @@
 public class RESTSessionCatalog extends BaseViewSessionCatalog
     implements Configurable<Object>, Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(RESTSessionCatalog.class);
-  private static final String DEFAULT_FILE_IO_IMPL = "org.apache.iceberg.io.ResolvingFileIO";
-  private static final String REST_METRICS_REPORTING_ENABLED = "rest-metrics-reporting-enabled";
-  private static final String REST_SNAPSHOT_LOADING_MODE = "snapshot-loading-mode";
-  // for backwards compatibility with older REST servers where it can be assumed that a particular
-  // server supports view endpoints but doesn't send the "endpoints" field in the ConfigResponse
-  static final String VIEW_ENDPOINTS_SUPPORTED = "view-endpoints-supported";
-  public static final String REST_PAGE_SIZE = "rest-page-size";
+
+  /**
+   * @deprecated will be removed in 2.0. Use {@link
```

Comment:
For consistency with other deprecations:
```suggestion
   * @deprecated will be removed in 2.0.0. Use {@link
```

Reply:
done